tutor/config.py

```diff
@@ -111,6 +111,12 @@ def get_user(root: str) -> Config:
         config = get_yaml_file(path)
     upgrade_obsolete(config)
     update_with_env(config)
+
+    extra_config: list[tuple[str, ConfigValue]] = []
+    extra_config = hooks.Filters.CONFIG_USER.apply(extra_config)
+    for name, value in extra_config:
```

Comment:
We have some syntactic sugar to write this more simply:

```
for name, value in hooks.Filters.CONFIG_USER.iterate():
    ...
```

Reply:
Updated to
```
def get_user(root: str) -> Config:
    """
    Get the user configuration from the tutor root.

    Overrides from environment variables are loaded as well.
    """
    convert_json2yml(root)
    path = config_path(root)
    config = {}
    if os.path.exists(path):
        config = get_yaml_file(path)
    upgrade_obsolete(config)
    update_with_env(config)

    for name, value in hooks.Filters.CONFIG_USER.iterate():
        config[name] = value
    
    return config
    ```
    
    That's much cleaner, thanks for the tip!